core/src/main/res/drawable/bg_pill_translucent.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android" >
+    <solid android:color="#D2404040"/>
+    <corners android:radius="18dp" />
+    <size android:height="14dp"
+        android:width="28dp"/>
```

Comment:
This will lead to problems when changing the font size of the device. Maybe have a look at `bg_pill.xml`.